client/test/e2e/journal/journal.spec.js

```diff
@@ -17,17 +16,51 @@ describe('Posting Journal Core', function () {
 
   /** @const */
   var initialTransactionRows = 1;
-  
-  // this will be run before every single test ('it') - navigating the browser 
-  // to the correct page.
-  beforeEach(function () { 
+
+  it('displays initial transactions loaded from database', function () {
+
     browser.get(route);
-  });
 
-  it('displays initial transactions loaded from database', function () { 
-    var journal = new JournalCorePage();
+    browser.waitForAngular();
+
+    var journal = new JournalPage();
     
     // @todo Test updated to test current system, updated with final mock transaction algorithm
-    expect(journal.totalRows()).to.eventually.be.above(initialTransactionRows);
+    expect(journal.getTotalRows()).to.eventually.be.above(initialTransactionRows);
+  });
+
+  it('reset the number of visible columns to default', function (){
+
+    var defaultVisibleColumnNumber = 6;
+
+    browser.get(route);
+
+    browser.waitForAngular();
```

Comment:
According to the protractor documentation [cited here](http://stackoverflow.com/questions/30882671/protractor-where-to-use-browser-waitforangular), there shouldn't be a need to call `browser.waitForAngular()`.  In the future, please avoid using this function.